tests/pytorch_pfn_extras_tests/dynamo_tests/test_compile.py

```diff
@@ -0,0 +1,76 @@
+import sys
+
+import pytest
+import pytorch_pfn_extras as ppe
+import torch
+from pytorch_pfn_extras import testing
+
+
+class _DummyModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.linear = torch.nn.Linear(10, 10)
+
+    def forward(self, x):
+        return self.linear(x).sum() * torch.tensor(5.0)
```

Comment:
torch.compile can also compile Module such as the following by partitioning the graph. However, it appears challenging to handle such Module using the ppe.compile interface. Is this the expected behavior?

```
class _DummyModule(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.linear = torch.nn.Linear(10, 10)

    def forward(self, x):
        if x.sum() < 0:
            return self.linear(x).sum() * torch.tensor(5.0)
        else:
            return self.linear(x).sum() * torch.tensor(1.0)
```

Reply:
Added a test and restricted this case!